Source/WebCore/platform/graphics/coretext/FontPlatformDataCoreText.cpp

```diff
@@ -183,4 +184,16 @@ String FontPlatformData::familyName() const
     return { };
 }
 
+FontPlatformData FontPlatformData::cloneWithSize(const FontPlatformData& source, float size)
+{
+    FontPlatformData copy(source);
+    copy.m_size = size;
+    // We need to reinitialize the instance, because the difference in size
+    // necessitates a new scaled font instance.
```

Comment:
Comment should be on one line. I don’t understand why we need this comment. Clearly we are allocating a new `CTFont` because one of the attributes is different, and clearly that is the size. If you think it’s not obvious, then I suppose we could write a comment: "Need a new CTFont with the new size". I would not use the phrase "reinitialize the instance", or the word "necessitates".